.gitignore

```diff
@@ -1,12 +1,12 @@
 katex.tar.gz
 katex.zip
+build
 node_modules
 npm-debug.log
 debug.log
 last.png
 diff.png
 /.npm-install.stamp
-/dist/
```

Comment:
Is this because people shouldn't be running `yarn dist` locally?

Reply:
@kevinbarabash They probably shouldn't and won't be needed.